README.md

```diff
@@ -0,0 +1,99 @@
+# CassSchema
+
+A gem for managing multiple cassandra schemas across multiple clusters. CassSchema supports loading a table schema from scratch, as well as running a migration to apply a change to the schema. Unlike some other database migration tools, there is no stored state about which migrations have and have not been run. -- a migration is simply a CQL statement to be run against the database.
+
+CassSchema operations apply to multiple 'datastores'. A datastore is a cluster, keyspace pair, so there may be multiple schemas for a single cluster, but only a single schema for a given cluster plus keyspace.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'cass_schema'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install cass_schema
+
+## Usage
+
+Before usage, CassSchema must be initialized with a set of datastore objects, as well as a base directory where the schema definitions live:
+
+```ruby
+CassSchema::Runner.datastores = CassSchema::YamlHelper.datastores('my/path/to/config.yml')
+CassSchema::Runner.schema_base_path = 'my/path/to/schema/root'
+```
+
+Here the datastores are a list of `CassSchema::DataStore` objects. These may be build manually, or loaded from a yaml file using the `CassSchema::YamlHelper`. CassSchema is intentionally agnostic about the source of these datastores.
+
+The schema_base_path is a directory where the schema definitions and migrations live. The structure of this directory should be:
```

Comment:
if this is being open sourced it might be nice to 'recommend place in rails to live'